src/com/palmergames/bukkit/towny/command/PlotCommand.java

```diff
@@ -165,6 +172,33 @@ public boolean parsePlotCommand(Player player, String[] split) throws TownyExcep
 					} else {
 						player.sendMessage(TownySettings.getLangString("msg_err_empty_area_selection"));
 					}
+				} else if (split[0].equalsIgnoreCase("eviction")) {

```

Comment:
I think this command should be 'evict' rather than 'eviction', it's shorter and it's more accurate as to what the user is doing (eviction is happening to the person on the other end.)